example/spec/parallel/addUniqueGranuleId/AddUniqueGranuleIdSpec.js

```diff
@@ -59,6 +59,10 @@ describe('The Add Unique Granule Id Task ', () => {
               ],
             },
           },
```

Comment:
The testSetup method takes `configOverride`, you can have multiple tests with different configOverride.task_config. A separate test file isn't needed.